miso-web/src/main/webapp/scripts/form_requisition.js

```diff
@@ -24,6 +24,21 @@ FormTarget.requisition = (function () {
       return Urls.ui.requisitions.edit(requisition.id);
     },
     getSections: function (config, object) {
+      /* adding the currently assigned assay to the requisition to the options of assays to choose from, because it
+           may not be part of the set of mutual assays between all the requisitioned samples' assays. */
+      var assayDropdown = [];
+      for (var i = 0; i < config.assayIdDropdownOptions.length; i++) {
+        var holder = Constants.assays.filter(function (x) {
```

Comment:
Can use `Array.find` instead of `filter` for finding a single item, but we also have a `Utils.array.findUniqueOrThrow` (`lims.js`) that works better here